packages/components/src/components/chat/WelcomeMessageV2.vue

```diff
@@ -14,18 +14,23 @@
         </ul>
       </div>
     </div>
-    <div class="welcome-message-dynamic-placeholder" v-else>Analyzing workspace...</div>
+    <v-skeleton-loader class="welcome-message__loading" v-else />
```

Comment:
I noticed you dropped the cue "Analyzing workspace..."

Reply:
Typically this style of loading element does not include any text. Generally it only takes a few seconds to complete and reveal it's purpose.